data/data/rhcos-stream.json

```diff
@@ -1,9 +1,154 @@
+                "location": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-live-initramfs.aarch64.img",
+                "signature": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-live-initramfs.aarch64.img.sig",
+                "sha256": "6152443f864b3db3619e080299c4896f74a9a4ddb16ce590c998b756be90afcc"
+              },
+              "rootfs": {
+                "location": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-live-rootfs.aarch64.img",
+                "signature": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-live-rootfs.aarch64.img.sig",
+                "sha256": "e0cb04c04f28303f55043178de92f7c384e936a9f29b63a3d8cc62e4176472ee"
+              }
+            },
+            "raw.gz": {
+              "disk": {
+                "location": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-metal.aarch64.raw.gz",
+                "signature": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-metal.aarch64.raw.gz.sig",
+                "sha256": "127656b57942482c154e93ae13351f6397882f85a0659d5c11af7fa44481ab06",
+                "uncompressed-sha256": "6cef571c9ffe1a22fa1f4e49a6578fe79cebf15a4c19345e1f1e32416cb6bb07"
+              }
+            }
+          }
+        },
+        "qemu": {
+          "release": "48.84.202105102319-0",
+          "formats": {
+            "qcow2.gz": {
+              "disk": {
+                "location": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-qemu.aarch64.qcow2.gz",
+                "signature": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.8-aarch64/48.84.202105102319-0/aarch64/rhcos-48.84.202105102319-0-qemu.aarch64.qcow2.gz.sig",
+                "sha256": "9d9c6bacfa74f4f98aa2b2d5d06cca9cdb6e70165bb862c79b5d4353429fd0bd",
+                "uncompressed-sha256": "02b05650ae6fcadd43b30c64711e16b2b9a48f7eb02c10c324ce7dc6a94e024e"
+              }
+            }
+          }
+        }
+      },
+      "images": {
+        "aws": {
+          "regions": {
+            "ap-east-1": {
+              "release": "48.84.202105102319-0",
+              "image": "ami-0b28d982f7ff321f0"
+            },
+            "ap-northeast-1": {
+              "release": "48.84.202105102319-0",
+              "image": "ami-0ffc57b891e093438"
+            },
+            "ap-northeast-2": {
```

Comment:
Do you know what would need to happen to get an ARM AMI for ap-northeast-3?

Reply:
the issue is that the m6g instances are not available in certain regions (https://aws.amazon.com/about-aws/whats-new/2021/02/aws-graviton2-m6g-c6g-r6g-instances-available-additional-regions/), so there is no point pushing to them

Reply:
That makes sense. Are there no available ARM instances in certain regions? Or just no m6g instances?

Reply:
i think none of the m6g, m6gd or a1 instances...so basically i think none of the ARM instances. @yselkowitz  am i correct that the excluded regions had no ARM instances? not just m6g,m6gd ? .

Reply:
m6g is the most prevalent and widely available ARM instance category at this point, so it's basically one and the same.